tests/test_cli_contract_help.py

```
@@ -0,0 +1,213 @@
+    if not tokens or tokens[0] != "cmux":
+        raise RuntimeError(f"Probe must start with cmux: {probe.command}")
+
+    env = dict(os.environ)
+    for key in [
+        "CMUX_SOCKET_PASSWORD",
+        "CMUX_WORKSPACE_ID",
+        "CMUX_SURFACE_ID",
+        "CMUX_TAB_ID",
+    ]:
+        env.pop(key, None)
+    no_socket = f"/tmp/cmux-no-socket-{uuid.uuid4().hex}.sock"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-no-socket-"

(S108)